manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/how-to-create-a-new-sample-app.md

```diff
@@ -0,0 +1,43 @@
+# How to create a new sample app
+
+This guide provides step-by-step instructions on how one can create a new

```

Comment:
Hey @mrkop , @rashmihe 
Jfyi. No need to verify these steps.
The IRD team (formerly FIS) team wanted to have some high level documentation around how easily one can take the current sample app and build a new one, this document just provides some guidance on how to do that.